docs/release-notes/8.16.asciidoc

```diff
@@ -0,0 +1,116 @@
+[[release-notes-header-8.16.0]]
+== 8.16
+
+[discrete]
+[[release-notes-8.16.0]]
+=== 8.16.0
+
+[discrete]
+[[known-issue-8.16.0]]
+==== Known issues
+
+// tag::known-issue-189676[]
+[discrete]
+.Tags appear in Elastic AI Assistant's responses
+[%collapsible]
+====
+*Details* +
+On August 1, 2024, it was discovered that Elastic AI Assistant's responses when using Bedrock Sonnet 3.5 may include `<antThinking>` tags, for example `<search_quality_reflection>` ({kibana-issue}189676[#189676]).
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[breaking-changes-8.16.0]]
+==== Breaking changes
+
+* During shutdown, {kib} now waits for all the ongoing requests to complete according to the `server.shutdownTimeout` setting. During that period, the incoming socket is closed and any new incoming requests are rejected. Before this update, new incoming requests received a response with the status code 503 and body { "message": "{kib} is shutting down and not accepting new incoming requests" }.
+
+[discrete]
+[[features-8.16.0]]
+==== New features
+
+// * Introduces a new API route for listing Entity Store entities: `GET /api/entity_store/entities/list` ({kibana-pull}192806[#192806]).
```

Comment:
```suggestion
* Introduces Security's entity store as a technical preview ({kibana-pull}192806[#192806], {kibana-pull}191986[#191986]).
```

Reply:
Just a small suggestion to flesh out the description a bit:
"Introduces the entity store as a technical preview feature, which allows observed, imported, integrated, or uploaded entities to be stored persistently."